types/process.go

```diff
@@ -170,6 +170,7 @@ type Process struct {
 	VoteCount            *BigInt               `json:"voteCount"                cbor:"12,keyasint,omitempty"`
 	VoteOverwrittenCount *BigInt               `json:"voteOverwrittenCount"     cbor:"13,keyasint,omitempty"`
 	SequencerStats       SequencerProcessStats `json:"sequencerStats"           cbor:"16,keyasint,omitempty"`
+	MaxVotes             *BigInt               `json:"maxVotes,omitempty"       cbor:"17,keyasint,omitempty"`
```

Comment:
Do we need a bigInt here? I'd say that uint64 might be enough (18,446,744,073,709,551,615) and even uint32 too...

Reply:
I declared it as a uint256 in the contract because the voteCount and voteOverwrittenCount are both uint256s and are compared there. I therefore assumed that they should be the same type.